test/e2e/e2e_suite_test.go

```diff
@@ -317,7 +349,7 @@ func deployCAPIComponents(kindCluster kind.Cluster) {
 	fmt.Fprintf(GinkgoWriter, "Generating CAPI manifests\n")
 
 	// Build the manifests using kustomize
-	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api//config").Output()
+	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api/config").Output()
```

Comment:
ooh, I think we should probably be specifying a ref here, otherwise this is probably using master
```suggestion
	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api/config?ref=v0.5.0-rc.3").Output()
```